help/destinations/catalog/personalization/pega-profile.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Pega Profile Connector
+description: Use the Pega Profile Connector for S3 in Adobe Experience Platform to export full or incremental, or both, profile data to Amazon S3 cloud storage. In Pega Customer Decision Hub, data jobs can be scheduled in Customer Profile Designer to import profile data periodically from Amazon S3 storage. 
+---
+
+# Pega Profile Connector
+
+## Overview {#overview}
+
+Use the [!DNL Pega Profile Connector] in Adobe Experience Platform to create a live outbound connection to your [!DNL Amazon Web Services] (AWS) S3 storage to periodically export profile data to CSV files from Adobe Experience Platform into your own S3 buckets. In [!DNL Pega Customer Decision Hub], you can schedule data jobs to import this profile data from S3 storage to update the Pega Customer Decision Hub profile. This connector helps setup initial export of profile data and also helps sync up new profiles periodically into Pega Customer Decision Hub.  Having up-to-date data in Customer Decision Hub provides a better and updated view of your customer base for next-best-action decisioning.
```

Comment:
some minor comments: 

we can break the paragraph in two parts for easier consumption: 

Use the [!DNL Pega Profile Connector] in Adobe Experience Platform to create a live outbound connection to your [!DNL Amazon Web Services] (AWS) S3 storage to periodically export profile data to CSV files from Adobe Experience Platform into your own S3 buckets. In [!DNL Pega Customer Decision Hub], you can schedule data jobs to import this profile data from S3 storage to update the Pega Customer Decision Hub profile.

This connector helps set up the initial export of profile data and also helps sync up new profiles periodically into Pega Customer Decision Hub.  Having up-to-date data in Customer Decision Hub provides a better and updated view of your customer base for next-best-action decisioning.

also, there are some missing DNL tags. it would be great if you could add them, for consistency.